VERSION

```diff
@@ -82,17 +84,17 @@ date="Unreleased developer copy"
 # Version numbers are described in the Libtool current:revision:age
 # format.
 
-libmpi_so_version=30:0:10
+libmpi_so_version=31:0:11
```

Comment:
I agree with the libmpifh version change, but I do not see an `mpi.h` change that would cause this `libmpi_so_version` change.  I'm looking on the `v2.x` branch starting with the 62cd194e32d44242d29114779f9cda13c2a3e70b commit.  Are we looking at the same commit range?